dtm0/it/all2all/README.md

```diff
@@ -41,5 +41,48 @@ sudo ./all2all
 Hare is used for cluster bootstrap and shutdown. Firstly
 it needs to be patched by the hare.patch in current directory
 to make it able to generate clusted configuration that includes
-DTM0 service. After that it should be built and installed into
-the system (see [README.md](https://github.com/Seagate/cortx-hare/blob/main/README.md) in the root of Hare repo).
+DTM0 service. First copy hare.patch file to the hare folder and run 
+following command from hare directory for applying patch.
+```sh
+patch -p1 < hare.patch
+```
```

Comment:
When this patch is accepted by Hare and merged here, this step will not be needed.
Is that right?

Reply:
Yes I also think so. This may not be exac code which will go in hare. But for that it may take time so we will have to keep this step untill we have support for DTM0 in hare.